docker/Dockerfile

```diff
@@ -0,0 +1,108 @@
+FROM ubuntu:20.04
+
+ARG mediainforepo="https://mediaarea.net/repo/deb/repo-mediaarea_1.0-19_all.deb"
+ARG dovitoollink="https://github.com/quietvoid/dovi_tool/releases/download/1.4.6/dovi_tool-1.4.6-x86_64-unknown-linux-musl.tar.gz"
+ARG hdr10plustoollink="https://github.com/quietvoid/hdr10plus_tool/releases/download/1.2.2/hdr10plus_tool-1.2.2-x86_64-unknown-linux-musl.tar.gz"
+ARG mp4boxlink="https://github.com/gpac/gpac.git"
+ARG mp4boxtag="v1.0.1"
+ARG dotnetlink="https://download.visualstudio.microsoft.com/download/pr/546d50b2-d85c-433f-b13b-b896f1bc1916/17d7bbb674bf67c3d490489b20a437b7/dotnet-runtime-5.0.15-linux-x64.tar.gz"
+ARG pgs2srtlink="https://github.com/Tentacule/PgsToSrt/releases/download/v1.4.2/PgsToSrt-1.4.2.zip"
+ARG tesseractlink="https://github.com/tesseract-ocr/tessdata.git"
+
+COPY /dvmkv2mp4 /usr/local/bin
```

Comment:
One pro tip for future this should be last command in dockerfile, to avoid redoing wholething when only dvmkv2mp4 script changes :)

Reply:
Exactly, I also noticed this mistake lately. Especially as the ARM version needs to be built from source because most of the apps don't have ARM64 packages.
Feel free to reorder or I will do the correction once I submit the ARM version (it will take some time).